tools/chapel-py/src/method-tables/uast-methods.h

```diff
@@ -108,7 +108,7 @@ CLASS_BEGIN(AstNode)
                std::optional<chpl::Location>,
                auto loc = chpl::parsing::locateCurlyBracesWithAst(context, node);
                return getValidLocation(loc))
-  PLAIN_GETTER(AstNode, parenth_location, "Get the Location of the parentheses of this AstNode node",
+  PLAIN_GETTER(AstNode, paren_location, "Get the Location of the parentheses of this AstNode node",
                std::optional<chpl::Location>,
                auto loc = chpl::parsing::locateExprParenthWithAst(context, node);
```

Comment:
I suspect you want to rename `ExprParenth` here as well. This is generated via `frontend/include/chpl/uast/all-location-maps.h`; grepping for `ExprParenth` should give you all occurrences that need to change.